README.rst

```diff
@@ -10,22 +10,35 @@ AWS ParallelCluster - HPC for the Cloud
 .. |Version| image:: https://badge.fury.io/py/aws-parallelcluster.png
     :target: https://badge.fury.io/py/aws-parallelcluster
 
-AWS ParallelCluster is an AWS supported Open Source cluster management tool that makes it easy for you to deploy and
```

Comment:
This part has been written by the management so I think we should leave as it is.
It is just an high level description of the product.

Reply:
I understand your point about management but I think we can broaden this wording so ParallelCluster doesn’t look like a life-sciences-only service.  We can discuss this internally.  For now, I will revert back to the original description.